CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Add styling guide documentation for customizing SDK and UI component CSS.
 - Extend and enabled style customizing capabilities on the SDK components.
+- Add `MeetingSessionConfiguration` as a required param to `MeetingManager` join method.
+- Remove `MeetingSessionConfiguration` properties from `MeetingProvider` props.
```

Comment:
nit: Add a new line before the "changed".

Reply:
Done